docs/IUF_docs.go

```diff
@@ -794,14 +794,23 @@ const docTemplateIUF = `{
         },
         "Operations": {
             "type": "object",
+            "required": [
+                "local-path",
+                "name",
+                "static-parameters"
+            ],
             "properties": {
-                "localPath": {
+                "local-path": {
```

Comment:
We don't actually use local-path anywhere ... I am wondering if we should just remove it.

Reply:
I filed [CASMINST-5689](https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-5689)